tests/unit/test_base_command.py

```diff
@@ -38,6 +40,23 @@ class Test_base_command_logging(object):
     options
     """
 
+    def setup(self):
+        self.old_time = time.time
+        time.time = lambda: 1547704837.4
+        self.old_tz = getattr(os.environ, 'TZ', None)
+        os.environ['TZ'] = 'UTC'
+        if 'tzset' in dir(time):
```

Comment:
I would document why the "if" check is needed here. (I am curious myself.) Also, `hasattr()` is the better way to check this IMO (and in the subsequent checks below).